api/app/Models/User.php

```diff
@@ -594,22 +593,16 @@ public static function scopePublishingGroups(Builder $query, ?array $publishingG
     }
 
     /**
-     * Scope is IT
-     *
-     * Restrict a query by pool candidates that are for pools
-     * containing IT specific publishing groups
-     *
-     * @param  \Illuminate\Database\Eloquent\Builder  $query  The existing query being built
-     * @return \Illuminate\Database\Eloquent\Builder The resulting query
+     * Return users who have an available PoolCandidate in at least one IT pool.
      */
-    public static function scopeInITPublishingGroup(Builder $query)
```

Comment:
I think I'd rather see the `scopeInITPublishingGroup` left as is. There's no need for a scope which JUST calls two other scopes, with no additional information. I think overly nesting scopes like that may just obfuscate what's happening.

Reply:
I think making this two separate scopes would cause users with an unavailable IT candidate but available other candidate to be counted here.